ui2/src/api/api.js

```diff
@@ -37,19 +37,59 @@ export const login = async user => {
   }
 };
 
+export const getAuthToken = () => localStorage.getItem(AUTH_TOKEN);
+
 export const appInit = () => {
-  const token = localStorage.getItem(AUTH_TOKEN);
-  let loggedIn = false;
+  const token = getAuthToken();
   if (token) {
     axios.defaults.headers.common['Authorization'] = token;
-    loggedIn = true;
   } else {
     logout();
   }
 
-  return loggedIn;
+  return !!token;
```

Comment:
why do you need this?

Reply:
changed this after I request for this review, sorry.